### PR TITLE
Update DMCC docs

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Updated instructions on how to to use the EGI infrastructure for Disaster Mitigation and Agriculture community.

---

**Related issue :**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
@check-spelling-bot apply updates

**Comment by sebastian-luna-valero** [comment]
@check-spelling-bot apply [updates](https://github.com/EGI-Federation/documentation/actions/runs/4434393934/attempts/1).

**Review by glarocca — APPROVED** [review]
LGTM

File: .github/actions/spelling/expect.txt [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Expected "words" that aren't in the dictionary, one word per line.
 # Some arbitrary strings that are in test files that aren't really words.
 # They should be removed if the test are changed/removed.
-microk8s
+CMVFS
```
Comment (gwarf) [inline .github/actions/spelling/expect.txt:4]: CMVFS?

File: .github/actions/spelling/expect.txt [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 diracsgm
+microk8s
+mytoken
 reponame
 sgm
-mytoken
+Sinica
```
Comment (gwarf) [inline .github/actions/spelling/expect.txt:10]: This one should more likely go to allow.txt

File: .github/actions/spelling/expect.txt (hunk 1 above)
Comment (sebastian-luna-valero) [reply .github/actions/spelling/expect.txt:4]: Sorry: https://en.wikipedia.org/wiki/Transposed_letter_effect :)

**Review by enolfc — APPROVED** [review]
LGTM

**Comment by sebastian-luna-valero** [comment]
Thanks, I still need to add new content. I will confirm when the PR is ready for review.

**Comment by enolfc** [comment]
I forced the check links and seems to be good now

**Comment by enolfc** [comment]
> Thanks, I still need to add new content. I will confirm when the PR is ready for review.



> Thanks, I still need to add new content. I will confirm when the PR is ready for review.

Could you mark this as Draft then?

**Comment by sebastian-luna-valero** [comment]
@glarocca latest changes added. Please let us know your thoughts.

**Review by glarocca — APPROVED** [review]
LGTM

**Comment by gwarf** [comment]
@sebastian-luna-valero before we can merge, can you please add some content to the PR description?
It's what is used for the commit message when merging.

**Pull request merged**